src/Stripe.net/Entities/Invoices/Invoice.cs

```diff
@@ -364,16 +378,23 @@ internal object InternalSubscription
         [JsonProperty("tax")]
         public long? Tax { get; set; }
 
+        [Obsolete("Use DefaultTaxRates instead")]
         [JsonProperty("tax_percent")]
         public decimal? TaxPercent { get; set; }
 
         /// <summary>
-        /// If <c>billing_reason</c> is set to <c>subscription_threshold</c> this
+        /// If <code>billing_reason</code> is set to <code>subscription_threshold</code> this
```

Comment:
`<c>` was correct here, although this would be even better:
```suggestion
        /// If <see cref="BillingReason" /> is set to <c>subscription_threshold</c> this
```